src/select/types.js

```diff
@@ -132,8 +138,8 @@ export type PropsT = {
   id?: string,
   /** Defines if the comparison for a new creatable value should be case-insensitive. */
   ignoreCase?: boolean,
-  /** A ref to access the input element powering the select if it's a search select, or the container div if it isn't. */
-  controlRef?: ReactRefT<HTMLElement>,
+  /** An imperative handle exposing internal methods. To preserve backward compatibility. */
```

Comment:
This comment seems unfinished. To preserve backwards compatibility with what? The element? This type doesn't support an element.

Reply:
Good catch, thanks.